andela/hc-rogue-one
# [feature #156577846] Create departments and add checks to departments

#### What does this PR do?
Ensure that Teams can view all jobs in the team account dashboard listed together as one view

#### Description of Tasks to be complete
Create different departments, add checks to departments and filter departments so that when teams log in to the dashboard they should be able to view the jobs per department.

#### How should this be manually tested?
1: Install the project and install the requirements.txt
2: Start the server by running; python manage.py runserver, open localhost  http://localhost:8080 in the browser
3: Login into the site
4: From the dashboard click on the Departments link
5: On the department page create department and save it
6: Go to the create checks links, add the check name and department
7: Click on the Department dropdown menu and select department
8: You should be able to view checks for that particular department.
#### What are the relevant pivotal tracker stories?
#156577846
#### Checklist: I completed these to help reviewers

No? Unsure? -> configure your IDE, format the code and add the changes with git add . && git commit --amend
existing code that was well tested you do not have to add tests)

No? -> write tests and add them to this commit git add . && git commit --amend


No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.


No? Unsure? -> execute command git pull --rebase origin develop
